### PR TITLE
Refactor simplify classes in terms of verification data usage

### PR DESCRIPTION
**Pull request by pawelkisielewicz**
Resolves first part of https://codepoets.atlassian.net/browse/GF-76 - pass `verification_data` in constructor

### CODE REVIEW

File: golem/verificator/core_verifier.py [hunk 1]
```diff
@@ -25,24 +26,24 @@ def __init__(self):
         self.message = ""
         self.computer = None
 
-    def start_verification(self, verification_data):
+    def start_verification(self):
         self.time_started = datetime.utcnow()
-        self.subtask_info = verification_data['subtask_info']
-        if self._verify_result(verification_data):
+        self.subtask_info = self.verification_data['subtask_info']
```
Comment (dybi) [inline golem/verificator/core_verifier.py]: please mov it to `__init__()`

File: golem/verificator/rendering_verifier.py [hunk 2]
```diff
@@ -48,12 +49,12 @@ def _verify_result(self, results):
...
 
-    def simple_verification(self, verification_data):
-        if not super().simple_verification(verification_data):
+    def simple_verification(self):
+        if not super().simple_verification():
             return False
 
-        subtask_info = verification_data['subtask_info']
-        results = verification_data['results']
+        subtask_info = self.verification_data['subtask_info']
```
Comment (dybi) [inline golem/verificator/rendering_verifier.py:56]: once it is in `CoreVerivier.__init__()`, it can be deleted from here

**Review by dybi — COMMENTED** [review]
Generally, I would suggest not to keep whole `verification_data`, but to use it to initialize
 ```python
self.subtask_info = verification_data['subtask_info']
        self.resources = verification_data['resources']
        self.results = verification_data['results']`
```

File: golem/verificator/core_verifier.py (hunk 1 above)
Comment (pawelkisielewicz) [reply golem/verificator/core_verifier.py]: done

File: golem/verificator/rendering_verifier.py (hunk 2 above)
Comment (pawelkisielewicz) [reply golem/verificator/rendering_verifier.py:56]: done

File: tests/apps/blender/task/test_blenderrendertask.py [hunk 3]
```diff
@@ -138,13 +138,13 @@ def test_computation_failed_or_finished(self, mock_dtt):
...
 
-        def verification_finished1(verification_data):
+        def verification_finished1():
             result = {'reference_data': None,
                       'message': "",
                       'time_started': None,
                       'time_ended': None,
                       'extra_data': {}}
-            result['extra_data']['results'] = verification_data['results']
+            result['extra_data']['results'] = list(self.bt.results.values())[0]
```
Comment (dybi) [inline tests/apps/blender/task/test_blenderrendertask.py:152]: it looks like the type was changed here. Can you explain?

**Review by dybi — CHANGES_REQUESTED** [review]
the tests are failing - you haven't changed `TestBlenderVerifier` tests;
please fetch and merge `origin/verifier-cleanup/master` into your branch and make sure tests and lint pass
NOTE:
don't forget to run "slowtests": `pytest   --runslow tests/golem/verificator/`

File: tests/apps/blender/task/test_blenderrendertask.py (hunk 3 above)
Comment (pawelkisielewicz) [reply tests/apps/blender/task/test_blenderrendertask.py:152]: no, the type is the same. It was `string` and is `string` now

**Review by dybi — APPROVED** [review]

**Pull request merged**